docs/cephfs-snapshot-backed-volumes.md

```diff
@@ -0,0 +1,59 @@
+# Provisioning and mounting CephFS snapshot-backed volumes
+
+Snapshot-backed volumes allow CephFS subvolume snapshots to be exposed as
+regular read-only PVCs. No data cloning is performed and provisioning such
+volumes is done in constant time.
+
+For more details please refer to
+`docs/design/proposals/cephfs-snapshot-shallow-ro-vol.md`.
```

Comment:
add link for `docs/design/proposals/cephfs-snapshot-shallow-ro-vol.md` for easy navigation